framework/web/ErrorHandler.php

```diff
@@ -219,7 +219,10 @@ public function renderFile($_file_, $_params_)
         $_params_['handler'] = $this;
         if ($this->exception instanceof ErrorException || !Yii::$app->has('view')) {
             ob_start();
-            ob_implicit_flush(false);
+            // somehow implicit flush doesn't work for hhvm when handling error
+            if (!defined('HHVM_VERSION')) {
+                ob_implicit_flush(false);
+            }
```

Comment:
Well, it solves the issue, but it doesn't have any stacktraces. The output with only above block give me this (more or less):
```
HHVM Fatal Error – yii\base\ErrorException

syntax error, unexpected '}'
2. yii\base\ErrorHandler::handleFatalError()

// context variable
$_SERVER = [
...
```

But that output only shows up after commenting out https://github.com/yiisoft/yii2/blob/master/framework/base/View.php#L323 and https://github.com/yiisoft/yii2/blob/master/framework/base/View.php#L484

Meanwhile the whole changes give me this output (more or less):
```
HHVM Fatal Error – yii\base\ErrorException

syntax error, unexpected '}'
1. in /opt/application/apps/desktop/controllers/HomeController.php at line 28 {
    public $layout = 'basic';
 
    /**
     * Display the index page.
     */
    public function actionIndex()
    {
        return $this->render('index')
    }
}
2. in /opt/application/vendor/yiisoft/yii2/BaseYii.php – include(...) at line 288
3. yii\BaseYii::autoload('desktop\controllers\HomeControll...')
4. in /opt/application/vendor/yiisoft/yii2/base/Module.php – class_exists('desktop\controllers\HomeControll...') at line 563
5. in /opt/application/vendor/yiisoft/yii2/base/Module.php – yii\base\Module::createControllerByID('home') at line 521
6. in /opt/application/vendor/yiisoft/yii2/base/Module.php – yii\base\Module::createController('index') at line 449
7. in /opt/application/vendor/yiisoft/yii2/web/Application.php – yii\base\Module::runAction('home/index', []) at line 84
8. in /opt/application/vendor/yiisoft/yii2/base/Application.php – yii\web\Application::handleRequest(yii\web\Request) at line 375
9. in /opt/application/apps/desktop/web/index.php – yii\base\Application::run()

// context variable
$_SERVER = [
...
```